VultisigApp/VultisigApp/Chains/thorchain.swift

```diff
@@ -47,12 +81,11 @@ enum THORChainHelper {
         input.accountNumber = accountNumber
         input.sequence = sequence
         input.mode = .sync
-        // THORChain fee is 0.02 RUNE
         input.fee = CosmosFee.with {
-            $0.gas = 20000000
+            $0.gas = THORChainHelper.THORChainGas
```

Comment:
gas will not be changed , actually thorchain doesn't have gas , we can continue to hard code it

Reply:
Hardcoded.